src/MMLib.SwaggerForOcelot/Transformation/SwaggerJsonTransformer.cs

```diff
@@ -73,7 +73,7 @@ private string TransformSwagger(JObject swagger, IEnumerable<ReRouteOptions> reR
             return swagger.ToString(Formatting.Indented);
         }
 
-        private string TransformOpenApi(JObject openApi, IEnumerable<ReRouteOptions> reRoutes, string hostOverride = "/")
```

Comment:
Why you change default from `/` to `null`?

Reply:
Yes, that was an idea I had that in case I get to null I would delete the Servers node from the json but I definitely leave it with the default value you set since that functionality was finally not implemented. In the next pull request I will update it. Thanks for the corrections